CHANGES.md

```diff
@@ -35,6 +35,18 @@ To be released.
     `HashDigest<SHA256>?`.  [[#3446], [#3447]]
  -  (Libplanet.Explorer) Added `TxResult.OutputState` of type
     `HashDigest<SHA256>?`.  [[#3446], [#3447]]
+ -  (Libplanet.Explorer) Added `offsetStateRootHash` of type
+    `HashDigest<SHA256>?` argument for `StateQuery.states` field.
+    [[#3448], [#3449]]
+ -  (Libplanet.Explorer) Added `offsetStateRootHash` of type
+    `HashDigest<SHA256>?` argument for `StateQuery.balance` field.
+    [[#3448], [#3449]]
+ -  (Libplanet.Explorer) Added `offsetStateRootHash` of type
+    `HashDigest<SHA256>?` argument for `StateQuery.totalSupply` field.
+    [[#3448], [#3449]]
+ -  (Libplanet.Explorer) Added `offsetStateRootHash` of type
+    `HashDigest<SHA256>?` argument for `StateQuery.validators` field.
+    [[#3448], [#3449]]
```

Comment:
I think we might need at least one test case where query is made with `offsetStateRootHash` instead of `offsetBlockHash`.

Reply:
https://github.com/planetarium/libplanet/pull/3449/commits/7ae0d0a4ba31e80e8540d117c8d0a3600048679d added.